deliveries/identity_directory-milestone_2.md

```diff
@@ -0,0 +1,14 @@
+# Milestone Delivery :mailbox:
+
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/General-Grants-Program/blob/master/grants/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** https://github.com/w3f/Open-Grants-Program/blob/master/applications/substrate-identity-directory.md
+* **Milestone Number:** 2
+
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 1. | Implement logic for sending tokens | https://github.com/Shard-Labs/identity-directory/blob/main/src/store/actions.ts#L250 | Implement logic for sending tokens; retrieve balance, parse inputs, display transaction fee, create the transfer transaction. |
```

Comment:
* There is no unit tests for that functionality since we are only calling external module to send tokens. 
* Fixed now
* Send token function sends correct value of tokens (tested)